src/Microsoft.Identity.Client/AppConfig/AuthorityInfo.cs

```diff
@@ -35,7 +35,7 @@ internal class AuthorityInfo
         public AuthorityInfo(
             AuthorityType authorityType,
             string authority,
-            bool validateAuthority = false)
+            bool validateAuthority)
         {
             AuthorityType = authorityType;
             ValidateAuthority = validateAuthority;
```

Comment:
>ValidateAuthority [](start = 12, length = 17)

we should consider if the authorityType is B2C to simply set it to false.

Reply:
done.